Conceptpower+Spring/src/main/java/edu/asu/conceptpower/servlet/web/ConceptEditController.java

```diff
@@ -258,6 +267,47 @@ public ModelAndView confirmEdit(@PathVariable("id") String id, HttpServletReques
         return new ResponseEntity<String>(buildJSON(synonymList, false, true), HttpStatus.OK);
     }
 
+    @RequestMapping(method = RequestMethod.GET, value = "conceptEdit/conceptSearch")
+    public @ResponseBody ResponseEntity<Object> searchConcept(@RequestParam("concept") String concept,
```

Comment:
Doesn't this method do the same thing as the method that is called when creating a new concept? Can't you reuse, or combine these two?